RecoLocalCalo/HGCalRecProducers/plugins/TEST_HeterogeneousESProduct.cc

```diff
@@ -0,0 +1,55 @@
+#include "RecoLocalCalo/HGCalRecProducers/plugins/TEST_HeterogeneousESProduct.h"
```

Comment:
Test files (guessed from the name) should be placed into a separate dynamic library in `test` directory.